kube/minkube/services/storage-service.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kernelci-api-storage
+spec:
+  selector:
+    app: kernelci-api-storage
+  ports:
+    - protocol: TCP
+      port: 80
```

Comment:
```suggestion
      port: 8002
```

Reply:
Ditto - I think this should be 80

Reply:
Done, @gctucker I think it makes more sense to do what @JenySadadia is suggesting as they are just dummy placeholder ports, so the actual thing is still intact with the container, so I have made the changes accordingly. It has somewhat increased the visibility as well.